matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.css

```diff
@@ -475,6 +475,15 @@ td[name="contact"] {
   white-space: normal;
   line-height: 12px;
 }
+.xTooltip {
+  white-space: pre-wrap;
+  white-space: -moz-pre-wrap;
```

Comment:
The standard value is well supported, there is no need to use the prefixed versions.